.github/workflows/stale.yml

```
@@ -17,7 +17,7 @@ jobs:
           days-before-issue-close: 7
           only-labels: "status: needs information"
           stale-issue-label: "stale"
-          stale-issue-message: "This issue is stale because it has been open for 14 days with no activity."
-          close-issue-message: "This issue was closed because it has been inactive for 7 days since being marked as stale."
+          stale-issue-message: "This issue is stale because requested follow-up information was not provided for 14 days."
+          close-issue-message: "This issue was closed because follow-up information has not been provided for 7 days since being marked as stale."
```

How about mention the label too?

```suggestion
          stale-issue-message: "This issue is stale because it has the `status: needs information` label and requested follow-up information was not provided for 14 days."
          close-issue-message: "This issue was closed because it has the `status: needs information` label and follow-up information has not been provided for 7 days since being marked as stale."
```